test/dummy.test.js

```diff
@@ -3,14 +3,8 @@ const {assert} = suitest;
 
 describe('My super cool test', () => {
 	before(async() => {
-		try {
-			// Start test
-			await suitest.startTest('my-super-cool-test');
-		} catch (e) {
-			console.error(e);
-			// Mocha wouldn't stop if just throw an error here
-			process.exit(1);
-		}
+		// Start test
+		await suitest.startTest('my-super-cool-test');
```

Comment:
@AntonSBilan , did you tried to make this line fail? Is it really not needed anymore?

Reply:
@andriichumak I tried throw error inside before block, and in this case mocha exits

Reply:
OK, please try as well with unhandled Promise rejection. Maybe they fixed it in new Mocha version, but when this demo was created there was a problem with it.

Reply:
Promise.reject(''); causes process exit as well

Reply:
I tried to throw error in startTest chain explicitly and mocha keeps running.

Reply:
ok, its works, probably tested with old version